faqs.rst

```diff
@@ -0,0 +1,27 @@
+.. _faqs:
+
+==========
+FAQs
+==========
+Some frequently asked questions about flux and their answers.
+
+.. _flux-run-mac:
+
+-----------------------
+Does flux run on a mac?
+-----------------------
+
+Not yet. We have an open issue_ on GitHub tracking the progress towards the goal of natively compiling on a mac. In the meantime, you can use Docker, see: :ref:`quickstart`.
```

Comment:
For external links, I think inline links are probably preferred to avoid name collisions on the labels.

```suggestion
Not yet. We have an open `issue <https://github.com/flux-framework/flux-core/issues/2892>`_ on GitHub tracking the progress towards the goal of natively compiling on a mac. In the meantime, you can use Docker, see: :ref:`quickstart`.
```